connect_deprish_to_ferc1.py

```diff
@@ -31,57 +35,80 @@ def __init__(self, file_path_mul, file_path_steam_ferc1,
             file_path_deprish_eia (path-like): file path to the table
                 connecting the deprecation records to the EIA master unit list
         """
-        self.file_path_mul = file_path_mul
-        self.file_path_steam_ferc1 = file_path_steam_ferc1
-        self.file_path_ferc1_eia = file_path_ferc1_eia
-        self.file_path_deprish_eia = file_path_deprish_eia
-
+        # TODO: This is a bit of a placeholder riht now. I'd like to make
+        # functions like the get_master_unit_list_eia for each of these
+        # components. Right now, the pickled outputs are expected to be there.
         self.plant_parts_eia_raw = (
-            make_plant_parts_eia.get_master_unit_list_eia(self.file_path_mul))
+            make_plant_parts_eia.get_master_unit_list_eia(file_path_mul))
+        # right now we need both the steam table and the ferc1_eia connection
+        # because mostly
         self.steam_cleaned_ferc1_raw = pd.read_pickle(
-            self.file_path_steam_ferc1, compression='gzip')
+            file_path_steam_ferc1, compression='gzip')
         self.connects_ferc1_eia_raw = pd.read_pickle(
-            self.file_path_ferc1_eia, compression='gzip')
+            file_path_ferc1_eia, compression='gzip')
         self.connects_deprish_eia_raw = pd.read_pickle(
-            self.file_path_deprish_eia, compression='gzip')
+            file_path_deprish_eia, compression='gzip')
 
     def _prep_plant_parts_eia(self):
-        self.plant_parts_eia = self.plant_parts_eia_raw.reset_index()
+        self.plant_parts_eia = (
+            self.plant_parts_eia_raw.reset_index()
+            .pipe(pudl.helpers.cleanstrings_snake, ['record_id_eia']))
+
+    def _prep_steam_cleaned_ferc1(self):
+        self.steam_cleaned_ferc1 = (
+            self.steam_cleaned_ferc1_raw.reset_index()
+                .pipe(pudl.helpers.convert_to_date))
```

Comment:
I moved steam cleaned to be one of the needed inputs for _prep_connects_ferc1_eia, bc I wanted to use `connects_ferc1_eia` as "the" ferc table to connect depreciation to instead of steam. This is more consistent - we are using the `connects_deprish_eia` as "the" depreciation table as well.